R/uniDimBayesianOutputTables.R

```diff
@@ -223,6 +233,7 @@
 
     df <- data.frame(statistic = opts[idxSelected], prior = probsPrior, posterior = probsPost)
     probTable$setData(df)
+    probTable$addFootnote(note)
```

Comment:
shouldn't you check here if `note != ""`?

Reply:
Well. The footnote works when `note = ""`, there is just no note then. So why would I check it?

Reply:
> So why would I check it?

Because it shows that you don't intend to set a footnote to a future reader. Now, you depend on some undocumented implementation detail. 

Also, I'm not seeing anywhere in https://github.com/jasp-stats/jaspResults/blob/8807c66fc14fcdedb6576ee17c92af16a7e3fb48/R/zzzWrappers.R#L590-L603 or https://github.com/jasp-stats/jaspResults/blob/8807c66fc14fcdedb6576ee17c92af16a7e3fb48/src/jaspTable.cpp#L1100-L1117 where the case `""` is explicitly ignored, so this could also be a bug that is interpreted as a feature and that might change in the future.

Reply:
Hmm okay. I was actually thinking that the object `note <- ""`is a character in R and thus it is fine to add it in a table footnote. I am doing this in several other instances, because I thought it was more handy than checking everytime if the footnote wasn't empty.